app/api/video_stream_moderators.py

```diff
@@ -20,6 +22,12 @@ def before_post(self, args, kwargs, data):
         if not has_access('is_coorganizer', event_id=stream.event_id):
             raise ForbiddenError({'pointer': 'user_id'}, 'Co-Organizer access required')
 
+    def after_create_object(self, video_stream_moderator, data, view_kwargs):
+        event = safe_query(
+            Event, 'id', video_stream_moderator.video_stream._event_id, 'event_id'
+        )
```

Comment:
No need to pass event as it can be queried via video_moderator

Reply:
means? I didnt understood, we can't access from video_moderator eveyrttime because it contains rooms also as we discussed in chat. @iamareebjamal

Reply:
You are accessing event from moderator here itself, so what will change if you move this line in the function itself?